### DISCUSSION TITLE
Error: Compiled in linux, but executable doesn't work.

### DISCUSSION THREAD

**Discussion by biotemon** [pawelsalawa/sqlitestudio#4077] [Q&A]
I compiled from source in Linux Ubuntu 20.04 LTS with no errors. Inside the output folder I found the `sqlitestudiocli ` file. I try double click it and I get a message "Could not open sqlitestudiocli Archive type not supported"  How can I fix it? 

Can you please provide a windows precompiled version? Worst comes worst I could run it in Linux using Wine. 

Thank you

**Comment by pawelsalawa** [comment]
Windows binaries are available from both homepage (https://sqlitestudio.pl - the big orange "Download" button at the center) and on GitHub's Releases page: https://github.com/pawelsalawa/sqlitestudio/releases

After compiling under Windows you need to build proper package (directory with proper dll files), because Windows looks for dependencies in local execution folder and then in PATH variable. If anything is missing, it throws error like you saw. Anyway, you would better just use ready binaries that I mentioned above. No need to fight with compiling.

Lastly, the `sqlitestudiocli` is Command Line Interface and it's very experimental (thus unstable).

You probably want to run SQLiteStudio.exe - which is the GUI application.
